pyuvdata/miriad.py

```diff
@@ -37,37 +37,44 @@ def _pol_to_ind(self, pol):
                              "polarization_array".format(pol=pol))
         return pol_ind
 
-    def read_miriad(self, filepath, correct_lat_lon=True, run_check=True,
-                    check_extra=True, run_check_acceptability=True, phase_type=None,
-                    antenna_nums=None, ant_str=None, bls=None,
-                    polarizations=None, time_range=None):
+    def read_miriad(self, filepath, antenna_nums=None, ant_str=None, bls=None,
+                    polarizations=None, time_range=None, read_data=True,
+                    phase_type=None, correct_lat_lon=True, run_check=True,
+                    check_extra=True, run_check_acceptability=True):
         """
         Read in data from a miriad file.
 
         Args:
             filepath: The miriad file directory to read from.
-            correct_lat_lon: flag -- that only matters if altitude is missing --
-                to update the latitude and longitude from the known_telescopes list
-            run_check: Option to check for the existence and proper shapes of
-                parameters after reading in the file. Default is True.
-            check_extra: Option to check optional parameters as well as required
-                ones. Default is True.
-            run_check_acceptability: Option to check acceptable range of the values of
-                parameters after reading in the file. Default is True.
-            antenna_nums: The antennas numbers to only read into the object.
+            antenna_nums: The antennas numbers to read into the object.
             bls: A list of antenna number tuples (e.g. [(0,1), (3,2)]) or a list of
                 baseline 3-tuples (e.g. [(0,1,'xx'), (2,3,'yy')]) specifying baselines
                 to keep in the object. For length-2 tuples, the  ordering of the numbers
                 within the tuple does not matter. For length-3 tuples, the polarization
-                string is in the order of the two antennas. If length-3 tuples are provided,
-                the polarizations argument below must be None.
+                string is in the order of the two antennas. If length-3 tuples are
+                provided, the polarizations argument below must be None.
             ant_str: A string containing information about what kinds of visibility data
                 to read-in.  Can be 'auto', 'cross', 'all'. Cannot provide ant_str if
                 antenna_nums and/or bls is not None.
             polarizations: List of polarization integers or strings to read-in.
                 Ex: ['xx', 'yy', ...]
             time_range: len-2 list containing min and max range of times (Julian Date) to read-in.
                 Ex: [2458115.20, 2458115.40]
+            read_data: Read in the visibility and flag data. If set to false,
+                only the basic header info and metadata (if read_metadata is True)
```

Comment:
read_metadata is only a thing for uvfits because uvfits essentially has 3 levels of things to read: header info, metadata and data, so you need two keyword arguments to tell it what to read beyond the header. Miriad only has 2 levels: metadata & data, so it only needs one keyword argument (if you call it at all it reads the metadata).

Reply:
Yes, that makes sense. Sorry for not being clear, I was just saying that the docstring has a reference to the `read_metadata` kwarg, so it probably needs to be clarified.

Reply:
Ah! Thanks for catching that and I'm sorry I didn't pay enough attention!